config/vufind/contentsecuritypolicy.ini

```diff
@@ -0,0 +1,32 @@
+; Settings for Content Security Policy header; you can learn more here:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+[CSP]
+; It is possible to use CSP in report-only mode. See report_uri setting below.
+; This setting could be set differently for each APPLICATION_ENV value (written as
+; array key below). Please note, that Whoops error handler (enabled in development
+; mode) does not show correctly when report_only is false.
+report_only[production] = false
+report_only[development] = true
+
+; The nonce (number used once) - unique number for each request, it is strongly
+; recommend to keep this setting on
+use_nonce = true
+
+; Directives, you can find available directives, on this page:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+; For evaluation of CSP you can use this tool provided by Google:
+; https://csp-evaluator.withgoogle.com/
+[Directives]
+default-src[] = "'none'"
+script-src[] = "'strict-dynamic'"
+script-src[] = "'unsafe-inline'"
+script-src[] = "http:"
```

Comment:
Should we suggest some best practices either here or in a new wiki page? For example, if I'm understanding correctly, many users might want to disable 'http:' if they're running secure sites, etc. I'm happy to establish a wiki page for this if that would be helpful (and then we could link to it from comments here).

Reply:
I do think the wiki page would be great for this, could you create one? 

I tried to make the setting as compatible with all kinds of installations and user agents as possible, so there is still room for improve the security

Reply:
I have created a placeholder page [here](https://vufind.org/wiki/administration:security:content_security_policy). Please feel free to fill in more details. I will also expand it further (with links to GitHub for the .ini file, etc.) after this PR is merged.